src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -8,432 +8,51 @@ import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
-			}
-			else {
-				ReviewHelper.validateDetachedSubmission(this.props.submission.id)
-				.then((response) => {
-					this.setState({
-						detachedAward: item,
-						validationFinished: true,
-						headerErrors: false,
-						jobResults: response
-					});
-				});
-			}
-		}
-		else if (data.jobs[0].job_status == 'finished') {
-			// don't run the check again if it's done
-			runCheck = false;
-
-			// display dowload buttons
-			// make a clone of the file's react state
-			const item = Object.assign({}, this.state.detachedAward);
-			item.status = "done";
-
-			ReviewHelper.validateDetachedSubmission(this.props.submission.id)
-				.then((response) => {
-					this.setState({
-						detachedAward: item,
-						validationFinished: true,
-						headerErrors: false,
-						jobResults: response
-					});
-				});
-		}
-
-		if (this.isUnmounted) {
-			return;
-		}
-
-		if (runCheck && !this.isUnmounted) {
-			// wait 5 seconds and check the file status again
-			window.setTimeout(() => {
-				this.checkFileStatus(this.props.submission.id);
-			}, timerDuration * 1000);
-		}
+			showMeta: false
+		})
 	}
 
 	render() {
-		let subTierAgencyIcon = <Icons.Building />;
-		let subTierAgencyClass = '';
-		if (this.state.agencyError) {
-			subTierAgencyIcon = <Icons.Building />;
-			subTierAgencyClass = ' error usa-da-form-icon';
-		}
-
-		let datePicker = null;
-		if (this.state.showDatePicker) {
-			let value = {
-				datePlaceholder: "Action",
-				endDate: this.state.detachedAward.endDate, 
-				label: "File D2: Financial Assistance",
-				startDate: this.state.detachedAward.startDate
-			}
-			datePicker = <DateRangeWrapper {...this.state} 
-								handleDateChange={this.handleDateChange.bind(this, "detachedAward")} 
-								hideError={this.hideError.bind(this)} 
-								showError={this.showError.bind(this)} 
-								value={value}
-								disabled={this.state.showValidationBox} />;
-		}
-
-		let uploadFilesBox = null;
-		if (this.state.detachedAward.valid && this.state.showUploadFilesBox) {
-			uploadFilesBox = <UploadDetachedFilesBox {...this.state} 
-								hideError={this.hideError.bind(this)}
-								showError={this.showError.bind(this)}
-								submission={this.props.submission}  
-								uploadFile={this.uploadFile.bind(this)} />;
-		}
-
-		let validationBox = null;
-		if(this.state.showValidationBox) {
-			const type = {
-				fileTitle: 'File D2: Award',
-				fileTemplateName: 'detached_award.csv',
-				requestName: 'detached_award',
-				progress: '0'
-			}
-			validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults} />;
-			if(!this.state.headerErrors && this.state.validationFinished) {
-				validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} />;
-			}
+		let content = null;
+		if(!this.state.showMeta){
+			content = <UploadDetachedFileValidation submission={this.props.submission} setSubmissionId={this.props.setSubmissionId.bind(this)} ></UploadDetachedFileValidation>
```

Comment:
Why is there a closing tag? Can we just do `content = <UploadDetachedFileValidation submission={this.props.submission} setSubmissionId={this.props.setSubmissionId.bind(this)} />`